cli/cmd/main.go

```diff
@@ -712,6 +793,70 @@ func DataCommand(c *cli.Context) error {
 	}
 
 	filter := &datapb.Filter{}
+	if err := createDataFilter(c, filter); err != nil {
+		return err
+	}
+
+	client, err := rdkcli.NewAppClient(c)
+	if err != nil {
+		return err
+	}
+
+	dataType := c.String(dataFlagDataType)
+	switch dataType {
+	case dataTypeBinary:
```

Comment:
@AaronCasas I think this is showing up as something that is added in this PR since I re-arranged into the helper function. I believe this is fine, but can you just double check that this is as intended?

Reply:
Ah this is because your `createDataFilter` should stop at

```
if start != nil || end != nil {
		filter.Interval = &datapb.CaptureInterval{
			Start: start,
			End:   end,
		}
	}
```

But from copying over too much, it continues with `client, err := rdkcli.NewAppClient` and the rest of this `client.BinaryData` code. If you remove that, that'll fix the helper and should show a normal diff in GH.

Reply:
Ahh thank you! I missed it